xmodule/capa/tests/test_capa_problem.py

```diff
@@ -732,3 +733,101 @@ def test_get_question_answer(self):
         # Ensure that the answer is a string so that the dict returned from this
         # function can eventualy be serialized to json without issues.
         assert isinstance(problem.get_question_answers()['1_solution_1'], str)
+
+    def test_get_grade_from_answers_with_student_answers(self):
```

Comment:
The essence of the changes in this PR is the introduction of new fields in the CAPA problem that allow for grading differently per a grading policy. These tests focus on which methods are called, how. Don't we need tests that prove that the newly added fields are being used consistently with the new policies? i.e., wouldn't you expect to have at least one test per grading policy, to ensure that calculated grades, per the new fields, are correct?

Reply:
These tests were added in the `test_capa_block.py` module. There is already a test for each grading method, and there is also a test when switching between grading methods. From [here](https://github.com/openedx/edx-platform/blob/f8349168ec376086285c1b26ea6e2d74bf459150/xmodule/tests/test_capa_block.py#L942-L1090) you can see them.
Would this be enough? @mariajgrimaldi @bszabo